### PR TITLE
Zeitlerj_Economy_Superaction

### PR DESCRIPTION
**Pull request by zeitlerj**
- This is a...
  - [x] Superaction    
  - [ ] Action
- [ x I have followed the Guidelines for Contributing as closely as possible.
- [x] I have thoroughly tested my (super)action and ensured that it works properly, even when used incorrectly.
- [x] My action is not a wrapper/alias for another command - it has an actual use.

--------------------------------------------------------------------------------

### CODE REVIEW

File: Economy/zeitlerj_economy/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Zeitlerj Economy Superaction
+
+Zeitlerj's Economy Superaction provides a powerful and customizable economy system for atlas users. 
+
+### Included features are:
+* **Customizable Suffix** Default setting is `cR`, can be changed using subcommands to any desired suffix, including emojis.
```
Comment (doddsy) [inline Economy/zeitlerj_economy/README.md]: Could you edit it to say "customizable currency suffix" or "customizable currency symbol"? People are stupid and might confuse suffix with prefix (I did for a split second not gonna lie)

**Comment by zeitlerj** [comment]
Could I have clarification on the balance issues related to the pot and additional?

**Comment by Cykreet** [comment]
> Could I have clarification on the balance issues related to the pot and additional?

Enabling and then claiming the pot, causes balance to be `Error: unexpected TEOF: EOF`.

**Comment by zeitlerj** [comment]
Alright, that I can fix, its trying to do math with a null value.

**Comment by Cykreet** [comment]
Checking balance upon importing the action returns `Error: parse error [1:7]: Expected EOFcR`.

**Comment by zeitlerj** [comment]
Could I get some clarification on if this is the normal "checkbal" subcommand, or the "admin checkbal" subcommand, or both?

**Comment by Cykreet** [comment]
Seems to working *mostly*, though there were some things that I thought should be improved:
- Markdown used when showing what the economy's symbol has been set to.
- Commands used to toggle certain things are called `enable`, could cause confusion.
- Being able to buy a role you already own.
- Toggling enable state of the economy pot just outputs `true` regardless if it's already enabled.

Through minimal testing I couldn't get the pot to increase or actually work.

**Comment by jam8608** [comment]
Closing this PR as it is inactive. If you want to open a new one, feel free.

**Pull request closed without merging**